packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -166,7 +164,7 @@ class TimelineData {
     }
 
     if (debugTimeline) {
-      debugHandledTraceEvents.write('${jsonEncode(event.json)},');
+      debugHandledTraceEvents.add(event.json);
       debugFrameTracking.writeln('Handling - ${event.json}');
```

Comment:
We should remove the debugFrameTracking to a separate flag. I expect it is only something that is useful for devtools authors debugging what went wrong.

Reply:
Both `debugHandledTraceEvents` and `debugFrameTracking` are only for devtools authors, which is why they are behind the same `debugTimelineFlag`. Maybe I should rename the trace events that will be downloaded with the export button. Having debug in that name may be misleading.